### ISSUE TITLE
Delta disk for ARM64 preview 22523?

### ISSUE THREAD

**Issue by dscho** [jeremyd2019/winautoconfig#3]
The current wiki has instructions and refers to a delta disk targeting 22000, but https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewARM64 no longer offers that build, only 22523.

Would it be possible to update the delta disk? Or even better: is there a way to avoid the delta disk that requires a specific Windows build?

**Comment by jeremyd2019** [comment]
The delta disk is really convenient for me, but not necessary.  The most critical part of that is getting the virtio drivers in there.  That was the idea behind the dataimage_arm64.wim and the winarmconfig.iso, but there's a bug that I haven't tracked down that makes things more difficult than they should have been.

**Comment by jeremyd2019** [comment]
Basically, I stopped getting new builds once I got the build that corresponded to Windows 11.

**Comment by jeremyd2019** [comment]
I was able to work around the issue I ran into where I had to use wimapply to deploy the "data image", and then use Windows DISM to inject the drivers.  Now both can be done with DISM, and the `winarmconfig.iso` is actually useful.

https://github.com/jeremyd2019/winautoconfig/releases/tag/20211222
> For other Windows images, mount both the Windows image and winarmconfig.iso on a Windows machine, or in WinPE (either 32-bit x86 or ARM64, *not* AMD64 because Microsoft doesn't support using AMD64 DISM against ARM64, and WinPE lacks WoW). Change drive/directory to the root of the ISO, and run deploy.cmd X:\ where X:\ is where the ARM64 Windows image is mounted. This will deploy the requisite files, and inject the drivers into the Windows install.

Does this make more sense?

**Comment by dscho** [comment]
> Does this make more sense?

Yes, thank you. I plan on pursuing the goal of a Windows/ARM64 setup again in January.
